openbb_terminal/alternative/realestate/landRegistry_view.py

```diff
@@ -0,0 +1,149 @@
+""" UK Land Registry View """
+__docformat__ = "numpy"
+
+import logging
+import os
+
+from openbb_terminal.decorators import log_start_end
+from openbb_terminal.alternative.realestate import landRegistry_model
+from openbb_terminal.helper_funcs import print_rich_table, export_data
+from openbb_terminal.rich_config import console
+
+logger = logging.getLogger(__name__)
+
+
+@log_start_end(log=logger)
+def display_estate_sales(postcode: str, limit: int, export: str = "") -> None:
```

Comment:
Actually I think it makes more sense to only have limit on the _view functions.  So  that the SDK gets all the data.  

So my request would be to remove the limits in model functions (get data) and only apply those in the views

Reply:
Hi, are you suggesting to remove the limit argument from the model completely?
The issue with this could then be we could return a large number of records from the model only to discard them in the view? Time to retrieve large number of records may also be an issue.
I was just going to add limit: int=25 to the view and leave the parameter in the model as limit: int

Reply:
So if the time will be an issue, then having it in the model is fine.

Reply:
I'm actually a little conflicted now. I think if you're using the SDK you should know that a limit is being applied and you're forced to enter one.
I would like to  leave the limit parameter in the model but without a default.

A default value already gets passed from the controller, I could also if required add the default value to the function declaration in the view.

Reply:
Having the default in the model makes it easier to use out of the box.  If I enter sdk.cmd( pars ), it should not expect a limit argument.  If someone wants to get more results, they can up the limit.